src/components/Grid/Grid.stories.tsx

```diff
@@ -10,34 +10,22 @@ export default {
     children: (
       <>
         <Grid.Item>
-          <div className="fpo u-margin-top-none u-margin-bottom-none">
-            Grid Item
-          </div>
+          <div className="fpo !mt-0 !mb-0">Grid Item</div>
```

Comment:
I think all the `!mt-0 !mb-0` class combos can be changed to `!my-0`.
```suggestion
          <div className="fpo !my-0">Grid Item</div>
```